package-lock.json

```diff
@@ -1385,21 +1498,23 @@
         "node": ">=0.10.0"
       }
     },
+    "node_modules/klaw-sync": {
+      "version": "6.0.0",
+      "resolved": "https://registry.npmjs.org/klaw-sync/-/klaw-sync-6.0.0.tgz",
+      "integrity": "sha512-nIeuVSzdCCs6TDPTqI8w1Yre34sSq7AkZ4B3sfOBbI2CgVSB4Du4aLQijFU2+lhAFCwt9+42Hel6lQNIv6AntQ==",
+      "dev": true,
+      "dependencies": {
+        "graceful-fs": "^4.1.11"
+      }
+    },
     "node_modules/less": {
       "version": "4.1.2",
       "resolved": "https://registry.npmjs.org/less/-/less-4.1.2.tgz",
       "integrity": "sha512-EoQp/Et7OSOVu0aJknJOtlXZsnr8XE8KwuzTHOLeVSEx8pVWUICc8Q0VYRHgzyjX78nMEyC/oztWFbgyhtNfDA==",
       "dev": true,
       "dependencies": {
         "copy-anything": "^2.0.1",
-        "errno": "^0.1.1",
-        "graceful-fs": "^4.1.2",
-        "image-size": "~0.5.0",
-        "make-dir": "^2.1.0",
-        "mime": "^1.4.1",
-        "needle": "^2.5.2",
         "parse-node-version": "^1.0.1",
-        "source-map": "~0.6.0",
```

Comment:
This is weird. When I check out this PR and run `npm i` then it modifies package-lock.json back.

![Screenshot 2023-04-28 at 14 22 48](https://user-images.githubusercontent.com/1235442/235146473-8088baeb-ca5e-4511-8aed-e249d37d04d1.png)

The NPM version I use is `7.5.2`.

Reply:
Looking at the error message the optional chaining is not supported by NodeJS before v13. My version is `v12.22.12`. I'll upgrade. But we should ensure the NodeJS dependency in package.json. It looks like it can be done with https://stackoverflow.com/questions/29349684/how-can-i-specify-the-required-node-js-version-in-package-json